vscode-extension/src/openinvscode.ts

```diff
@@ -104,8 +104,11 @@ async function setupSSHConfig(vmId: string, prKey: string) {
         console.debug(`stdout: ${stdout}`);
         console.debug(`stderr: ${stderr}`);
     });
+
     // save private key to file
-    const prkeyPath = `${process.env['HOME']}/.config/devbox/ssh/keys/${vmId}.vm.devbox-vms.internal`;
+    const prkeyDir = `${process.env['HOME']}/.config/devbox/ssh/keys/`;
+    await ensureDir(prkeyDir);
+    const prkeyPath = `${prkeyDir}/${vmId}.vm.devbox-vms.internal`;
```

Comment:
there's no double `//` here? `prkeyDir` has a trailing `/` already 🤔

Reply:
I'll drop the / from prKeyDir